nginx.conf

```diff
@@ -94,7 +94,8 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/managing/transferring-subgraph-ownership/$    $scheme://$http_host/docs/$1/managing/transfer-and-deprecate-a-subgraph/    permanent;
         rewrite ^/docs/en/substreams/(?!index\.).+$                                     https://substreams.streamingfast.io                                         permanent;
         rewrite ^/docs/en/firehose/(?!index\.).+$                                       https://firehose.streamingfast.io                                           permanent;
-
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/managing/transfer-and-deprecate-a-subgraph/$  $scheme://$http_host/docs/$1/managing/transfer-a-subgraph/                  permanent;
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/managing/transfer-and-deprecate-a-subgraph/$  $scheme://$http_host/docs/$1/managing/deprecate-a-subgraph/                 permanent;
```

Comment:
Looks like you're attempting to redirect the same URL (`/docs/[locale]/managing/transfer-and-deprecate-a-subgraph/`) to multiple URLs. That is not possible; only the first `rewrite` rule will take effect.

Reply:
@benface I need a little clarity for future redirects (lets chat)